base/docs/helpdb/Base.jl

```diff
@@ -881,7 +884,7 @@ trunc
 """
     unsafe_convert(T,x)
 
-Convert `x` to a value of type `T`
+Convert `x` to a C argument of type `T` where `x` is the return value of `cconvert(T, ...)`.
```

Comment:
Well, depend on what do you mean by user input. `x` is the input to this function, sure, but it's basically never what the user inputs. It must always be the return value of `cconvert`.